s2i/examples/nodejs-s2i.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TaskRun
+metadata:
+  name: s2i-nodejs-1
+spec:
+  serviceAccount: kbaig
```

Comment:
could we use a better service account name here? :sweat_smile: 

Would it be nice if we can put a sample service account file as well?